packages/dataviews/src/components/dataviews-view-config/index.tsx

```diff
@@ -251,32 +253,42 @@ function ItemsPerPageControl() {
 	);
 }
 
-function FieldItem( {
-	field,
+function BaseFieldItem( {
+	fieldId,
+	label,
+	subLabel,
 	isVisible,
 	isFirst,
 	isLast,
 	canMove = true,
+	canHide = true,
+	isInteracting = false,
```

Comment:
Why we need this?

Reply:
Because the action button that opens the menu popover is only visible if the mouse is over or if there is focus, on the item, when we open the menu the focus goes to the menu. In consequence the button that opens the menu would be hidden and the menu it self would be hidden.
To fix this issue we add a class that while the menu is open shows all the actions.
To test the wrong behavior and why the class is needed we temporarily remove the class.
I added a comment decoumenting this.